client/my-sites/domains/domain-management/edit/navigation/enhanced.jsx

```diff
@@ -509,6 +510,19 @@ class DomainManagementNavigationEnhanced extends React.Component {
 		);
 	}
 
+	getRedirectSettings() {
+		const { domain, selectedSite, translate } = this.props;
+
+		return (
+			<DomainManagementNavigationItem
+				path={ domainManagementRedirectSettings( selectedSite.slug, domain.name ) }
+				materialIcon="language"
+				text={ translate( 'Redirect settings' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **34** times:
`translate( 'Redirect Settings' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/use-the-new-design-for-site-redirects)